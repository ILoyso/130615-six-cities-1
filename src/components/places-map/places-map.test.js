import React from 'react';
import renderer from 'react-test-renderer';
import PlacesMap from './places-map.jsx';


const placeMock = [
  {
    coordinates: [52.3909553943508, 4.85309666406198],
    img: `apartment-01.jpg`,
    isPremium: true,
    price: 120,
    rating: 93,
    title: `Beautiful & luxurious apartment at great location`,
    type: `apartment`
  }
];


it(`PlacesMap renders correctly`, () => {
  const map = renderer
    .create(<PlacesMap
      city={`Amsterdam`}
      places={placeMock}
    />)
    .toJSON();

  expect(map).toMatchSnapshot();
});
