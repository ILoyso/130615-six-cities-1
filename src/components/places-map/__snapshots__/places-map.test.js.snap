// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlacesMap renders correctly 1`] = `
<section
  className="Class Name__map map"
>
  <div
    style={
      Object {
        "height": "100%",
      }
    }
  />
</section>
`;
